R/output-batches.R

```diff
@@ -19,17 +19,68 @@
 #' - `<lazyframe>$sink_batches()` returns `NULL` invisibly.
 #' - `<lazyframe>$lazy_sink_batches()` returns a new [LazyFrame].
 #' @examples
+#' # One usecase for this function is to export larger-than-RAM data to file
+#' # formats for which polars doesn't provide a writer out of the box.
+#' # The example below writes a LazyFrame by batches to a CSV file for the
+#' # sake of the example, but one could replace `write.csv()` by
+#' # `haven::write_dta()`, `saveRDS()`, or other functions.
```

Comment:
My guess is that Polars will try to use as much RAM as possible, so if we really want to write data larger than RAM, I'm worried you'll hit OOM the moment we export a polars DataFrame to R.
In other words, I think specifying the chunk size option is essential in such a real-world use case.

Reply:
Done in [453925d](https://github.com/pola-rs/r-polars/pull/1561/commits/453925d1278896f197174ce03de659490a772e48)